README.md

```diff
@@ -190,7 +189,7 @@ Saves the provided stats.
 - **GitStats** The `GitStats` object.
 
 ### `iterateDays(data, callback)`
-Iterate the days, calling the callback function for each day.
+Iterate through the days, calling the callback function on each day.
```

Comment:
Please update here the JSDoc comment too: https://github.com/IonicaBizau/git-stats/blob/59884d52c52f084a6bf52fd362ee19f64bdfeaef/lib/index.js#L133